cmd/state-offline-uninstaller/offline-uninstaller.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/installmgr"
+)
+
+const (
+	cmdUnInstall = "uninstall"
+)
+
+type Params struct {
+	sourcePath string
+	path       string
+}
+
+func newParams() *Params {
+	return &Params{path: "/tmp"}
+}
+
+func main() {
+	if os.Getenv("VERBOSE") == "true" {
+		logging.CurrentHandler().SetVerbose(true)
+	}
+
+	runErr := run()
+	if runErr != nil {
+		errMsg := errs.Join(runErr, ": ").Error()
+		if locale.IsInputError(runErr) {
+			logging.Debug("state-offline-uninstaller errored out due to input: %s", errMsg)
+		} else {
+			logging.Critical("state-offline-uninstaller errored out: %s", errMsg)
+		}
+
+		fmt.Fprintln(os.Stderr, errMsg)
+	}
+}
+
+func run() error {
+	args := os.Args
+
+	out, err := output.New("", &output.Config{
+		OutWriter: os.Stdout,
+		ErrWriter: os.Stderr,
+	})
+	if err != nil {
+		return errs.Wrap(err, "Could not initialize outputer")
+	}
+
+	p := primer.New(nil, out, nil, nil, nil, nil, nil, nil, nil, nil)
+
+	params := newParams()
+	cmd := captain.NewCommand(
+		path.Base(os.Args[0]), "", "", p, nil, nil,
+		func(ccmd *captain.Command, args []string) error {
+			out.Print(ccmd.UsageText())
+			return nil
+		},
+	)
+	cmd.AddChildren(
+		captain.NewCommand(
+			cmdUnInstall,
```

Comment:
As with the installer; the sub-command seems unnecessary. Just use the top-level command.